samples/ControlCatalog/MainView.xaml

```diff
@@ -60,6 +60,7 @@
       <TabItem Header="TextBlock"><pages:TextBlockPage/></TabItem>
       <TabItem Header="ToggleSwitch"><pages:ToggleSwitchPage/></TabItem>
       <TabItem Header="ToolTip"><pages:ToolTipPage/></TabItem>
+      <TabItem Header="Transform"><pages:TransformPage/></TabItem>
```

Comment:
I understand that the TransformPage in the directory does not look very good. But then where is it better to move it or just delete it?